src/components/classifier/SwipeCard.js

```diff
@@ -124,7 +122,6 @@ class SwipeCard extends Component {
                     style={[styles.image, styles.imageShadow]}
                     resizeMethod="resize" 
                     resizeMode="contain"
-                    loadingText={'Loading Subject'}
```

Comment:
Is there any reason this was removed? Is this message now displayed to the user elsewhere?

Reply:
Yes, that component actually doesn't take that prop. I think it got merged by accident.